modules/fedimint-ln-common/src/gateway_endpoint_constants.rs

```diff
@@ -0,0 +1,16 @@
+pub const ADDRESS_ENDPOINT: &str = "/address";
+pub const BACKUP_ENDPOINT: &str = "/backup";
+pub const BALANCE_ENDPOINT: &str = "/balance";
+pub const CONFIGURATION_ENDPOINT: &str = "/config";
+pub const CONNECT_FED_ENDPOINT: &str = "/connect_fed";
```

Comment:
This should be `/connect-fed`. I believe this is what's breaking backwards compatibility.